app/views/account/comments/create.js.erb

```diff
@@ -0,0 +1,6 @@
+<% if @result %>
+  $('.comments').prepend('<%= j render(partial: "show_comments", locals: {commentable: @commentable, comments: [@comment] } ) %>');
+  $('form')[0].reset();
```

Comment:
I don't like $('form')[0].reset();

find explicit comment form. because "0" form could be a different form in future